dhis-2/dhis-api/src/main/java/org/hisp/dhis/render/RenderFormat.java

```diff
@@ -33,5 +33,21 @@
 public enum RenderFormat
 {
     JSON,
-    XML
+    XML,
+    CSV;
+
+    public static boolean isJson( String format )
+    {
+        return JSON.name().equalsIgnoreCase( format );
+    }
+
+    public static boolean isXml( String format )
+    {
+        return XML.name().equalsIgnoreCase( format );
+    }
+
+    public static boolean isCsv( String format )
+    {
+        return CSV.name().equalsIgnoreCase( format );
+    }
```

Comment:
Having an enum isn't it more natural to have a single method that is used like this: `XML.isEqual(format)` ?